dash/development/_py_components_generation.py

```diff
@@ -122,7 +120,7 @@ def __repr__(self):
          for p in prop_keys
          if not p.endswith("-*") and
          p not in python_keywords and
-         p not in ['dashEvents', 'fireEvent', 'setProps']] + ['**kwargs']
+         p != 'setProps'] + ['**kwargs']
```

Comment:
Not sure if that was intentional but still present for the [R generation](https://github.com/plotly/dash/blob/32277157bd2874dcca63436959c43e4b1c7f227f/dash/development/_r_components_generation.py#L166)

Reply:
Sorry, will modify my code to match.